src/error.ts

```diff
@@ -206,8 +206,12 @@ export class MongoRuntimeError extends MongoDriverError {
  * @category Error
  */
 export class MongoBatchReExecutionError extends MongoRuntimeError {
-  constructor(message: string) {
-    super(message);
+  constructor(message?: string) {
+    if (message !== undefined) {
```

Comment:
I think it's unlikely that we'll have a use case for an empty string message, so, this can probably be shortened everywhere to just `super(message || '<default message>')`